### PR TITLE
AIM on CentOS 7 starting from Abiquo 4.0

### PR DESCRIPTION
**Pull request by nacx**

### CODE REVIEW

File: src/Aim_server.skeleton.cpp [hunk 1]
```diff
@@ -0,0 +1,233 @@
+// This autogenerated skeleton file illustrates how to build a server.
```
Comment (apuig) [inline src/Aim_server.skeleton.cpp]: unsure if this class is useful, if not perhaps could be in .gitignore

Comment (nacx) [inline src/Aim_server.skeleton.cpp]: I'm unsure too... @enricruiz ?

Comment (enricruiz) [inline src/Aim_server.skeleton.cpp]: yes, can be removed

Comment (nacx) [inline src/Aim_server.skeleton.cpp]: Done!

**Pull request merged**